spec/views/notifications/index.html.erb_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "notifications/index.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Do we need this file?

Reply:
I could see us going either way. We have a lot of pending tests that we haven't discussed. I think keeping the placeholder, and then having a "how do we want to deal with system specs vs. view specs" larger conversation, especially since it sounds like we're about to shift more towards UI stuff.